Sources/X509/CertificatePrivateKey.swift

```diff
@@ -63,7 +63,7 @@ extension Certificate {
             self.backing = .rsa(rsa)
         }
         
-        #if INCLUDE_SECURE_ENCLAVE_SUPPORT
+        #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
```

Comment:
can this be `canImport(Darwin)` or too broad?

Reply:
It can be, but Yim's patch here is just reverting the change back to what it was, which I quite like. We can do a follow-up PR for `#canImport(Darwin)`.